src/hooks/useInitializeCollectionDraft.ts

```diff
@@ -119,8 +123,14 @@ function useInitializeCollectionDraft() {
                         'workflows.collectionSelector.error.message.draftCreationFailed',
                 });
             }
+
+            setCollectionInitializationDone(true);
```

Comment:
This should cover setting to `true` when either we made a new draft or are needing to use a live spec.

Reply:
Even though I think we don't really _need_ to set to true when using a liveSpec it felt like it was in the spirit of this flag

![image](https://github.com/user-attachments/assets/34730f07-a229-4998-a230-5c8c901003af)